app/controllers/controls/views/gradients.js

```diff
@@ -93,9 +93,21 @@ function generateRandomColor() {
 }
 
 function handleGradientType(e) {
-  var isLinear = e.index == Gradient.LINEAR;
-  selectedGradient = e.index;
-  
+  var isLinear;
```

Comment:
`let`? Default to `false` ?

Reply:
Went with being consistent with the rest of the file. Mind if if update all of it?